sdk/servicebus/azure-servicebus/azure/servicebus/management/_xml_workaround_policy.py

```diff
@@ -50,10 +34,6 @@ def on_request(self, request):
             if b'<ns1:' in request_body:
                 request_body = request_body.replace(b'ns1:', b'')
                 request_body = request_body.replace(b':ns1', b'')
-            if b'<Value>' in request_body:
```

Comment:
Nice! (assuming these got fixed upstream?)

Reply:
Yes, it's fixed. So it can be removed here.